website/public/main.css

```diff
@@ -166,6 +166,18 @@ nav.breadcrumbs ol > li:not(:first-child)::before {
   content: "➤ ";
 }
 
+/* make "#" selected sections stand out */
+h4:target::before {
+  content: "👉"; /*TODO(😎): localize for different layout directions (e.g. rtl)*/
+}
```

Comment:
I think we should add a background to the `h4` instead. Maybe `rgba(0,0,0,0.1)` for light mode and `rgba(255,255,255,0.1)` for dark mode.